source/2020-07-29-ember-3-20-released.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Ember 3.20 Released
+author: Jen Weber, Jared Galanis
+tags: Releases, 2020, 3, 3.20, Version 3.x
+responsive: true
+---
+
+Today the Ember project is releasing version 3.20 of Ember.js, Ember Data, and Ember CLI. This release of Ember.js is an LTS (Long Term Support) candidate. LTS candidates prioritize stability over the addition of new features, and have an extended support schedule.
+
+This release kicks off the 3.21 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
+
+You can read more about our general release process here:
+
+- [Release Dashboard](http://emberjs.com/releases/)
+- [The Ember Release Cycle](http://emberjs.com/blog/2013/09/06/new-ember-release-process.html)
+- [The Ember Project](http://emberjs.com/blog/2015/06/16/ember-project-at-2-0.html)
+- [Ember LTS Releases](http://emberjs.com/blog/2016/02/25/announcing-embers-first-lts.html)
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 3.20
+
+Ember.js 3.20 is an incremental, backwards compatible release of Ember with bugfixes, performance improvements, and minor deprecations.
+
+Developers who upgrade their Ember apps to `v3.20` should also update `@glimmer/component` to `v1.0.1` to avoid a deprecation warning that comes from the `@glimmer/component` package.
```

Comment:
The deprecation is triggered by glimmer component 1.0.0, but actually lives in ember itself. If you think that distinction isn't super useful here, feel free to disregard.

```suggestion
Developers who upgrade their Ember apps to `v3.20` should also update `@glimmer/component` to `v1.0.1` to avoid a deprecation warning that comes from the `@glimmer/component` packages use of a private API.
```